SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005948	Nexon PC Bang
+SKILL_20151224_005949	Additional +30% Experience gained when defeating enemies{nl}Mark tax reduced by 20%{nl}Item listing count on market increased{nl}Train 3 attribute skills at same time{nl}Character Moving speed +3{nl}Stamina recovery doubled
+SKILL_20151224_005950	Book of Experience
+SKILL_20151224_005951	Additional +30% Experience gained when defeating enemies{nl}Stamina recovery doubled
+SKILL_20151224_005952	Enables you to equip the [Pistol] weapon type. You can also press the C button to use attack.
+SKILL_20151224_005953	[Enables you to equip [Cloth] armor. When equipped with 3 or more pieces of [Cloth] armor, increases your maximum SP and magic damage taken is decreased by 5%. When equipped with 4 [Cloth] armor pieces, your Magic Defense increases and magic damage taken is decreased by an additional 10%.
+SKILL_20151224_005954	[Enables you to equip [Leather] armor. When equipped with 3 or more pieces of [Leather] armor, increases your Evasion. When equipped with 4 [Leather] armor pieces, your Evasion increases from attributes by an additional 50%.
+SKILL_20151224_005955	Enables you to equip [Plate] armor. When equipped with 3 or more pieces of [Plate] armor, increases your maximum HP and physical damage taken is decreased by 5%. When equipped with 4 [Plate] armor pieces, your maximum Stamina increases and physical damage taken is decreased by an additional 10%.
+SKILL_20151224_005956	Stamina consumption cycle duration of companion is increased by 10 seconds per attribute level.
+SKILL_20151224_005957	If companion HP is less than 50%, activate [응급치료] to recover HP for 2 seconds. The duration of the recovery is increased by 1 second per attribute level. [응급치료] can't be used for another 10 minutes after activation
+SKILL_20151224_005958	Bash: Specialty
+SKILL_20151224_005959	Increases the damage dealt on an enemy with [Bash] by 35 per attribute level.
+SKILL_20151224_005960	Increases the maximum HP of your character by 5% per attribute level while [Swash Buckling] is active.
+SKILL_20151224_005961	Increases your physical defense by 10% per attribute level when [Guard] is active.
+SKILL_20151224_005962	Increases the character's evasion by 6% per attribute level while [Guardian] is active.
+SKILL_20151224_005963	Critical hit attack is increased depending on the physical attack when [Aggressor] is active. Additional value is increased by 5% per attribute level.
+SKILL_20151224_005964	Shield Shoving: Enhance
+SKILL_20151224_005965	Increases the damage dealt on an enemy with [Shield Shoving] by 1% per attribute level.
+SKILL_20151224_005966	Shield Bash: Enhance
+SKILL_20151224_005967	Increases the damage dealt on an enemy with [Shield Bash] by 1% per attribute level.
+SKILL_20151224_005968	Slithering: Additional Damage
+SKILL_20151224_005969	The enemies hit by [Slithering] is vulnerable to [Strike] for 10 seconds and increase the additional damage by 20% per attribute level.
```

Comment:
Enemies hit by [Slithering] become vulnerable to Strike-property attacks for 10 seconds, increases additional damage by 20% per attribute level.